recipes/cms-combine/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "cms-combine" %}
+{% set version = "10.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a6fecc8b4adc4e631425eeee7501d0258ab64ef36b24680696c2e6f8d38403e7
+  folder: HiggsAnalysis/CombinedLimit
+  patches:
+    - clang_warnings.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports: python
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - python # [build_platform != target_platform]
+    - cross-python_{{ target_platform }} # [build_platform != target_platform]
+  host:
+    - pip
+    - python
+    - root_base
+    - boost-cpp
+    - eigen
+    - vdt
+
+  run:
+    - python
+    - pandas
+    - gsl
+    - libboost
+
+test:
+  imports:
+    - HiggsAnalysis.CombinedLimit.PhysicsModel
+  commands:
+    - combine -h
+    - text2workspace.py -h
+
+about:
+  home: https://cms-analysis.github.io/HiggsAnalysis-CombinedLimit/
+  summary: 'A software tool used for statistical analysis within the CMS experiment'
+  description: |
+    Combine is a RooStats / RooFit - based software tool used for statistical analysis within the
+    CMS experiment. Note that while this tool was originally developed in the Higgs PAG, its usage
+    is now widespread within CMS. Combine provides a command-line interface to many different statistical
+    techniques, available inside RooFit/RooStats, that are used widely inside CMS.
+  license:  Apache-2.0
+  license_family: Apache
+  license_file: HiggsAnalysis/CombinedLimit/LICENSE
+  doc_url: https://cms-analysis.github.io/HiggsAnalysis-CombinedLimit/
+  dev_url: https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit
+
+extra:
+  recipe-maintainers:
+    - nsmith-
+    - kcormi
```

Comment:
@kcormi @anigamova FYI I have added you as maintainers.

Reply:
Note that they will both need to respond that they approve for the linting to pass.

Reply:
thanks @nsmith-, fine with me of course!